ksqldb-engine/src/main/java/io/confluent/ksql/schema/registry/SchemaRegistryUtil.java

```diff
@@ -89,6 +93,21 @@ public static void deleteSubjectWithRetries(
     }
   }
 
+  public static boolean subjectExists(
+      final SchemaRegistryClient srClient,
+      final String subject
+  ) {
+    try {
```

Comment:
Any reason we can't just call `getLatestSchema(...).isPresent()`?

Reply:
that's a good idea, I have to do some refactoring to make it work but i'll just do that.